README.md

```diff
@@ -2,7 +2,7 @@
 
 An implementation of [Amazon Ion Hash](http://amzn.github.io/ion-hash) in Go.
 
-[![Build Status](https://travis-ci.com/amzn/ion-hash-go.svg?branch=master)](https://travis-ci.com/amzn/ion-hash-go)
+[![build](https://github.com/amzn/ion-hash-go/workflows/Build/badge.svg)](https://github.com/amzn/ion-hash-go/actions)
```

Comment:
maybe link to https://github.com/amzn/ion-hash-go/actions?query=workflow%3ABuild ?